### DISCUSSION TITLE
빌드잇 유니티 Map Loader 관련

### DISCUSSION THREAD

**Discussion by heavybowgun** [naverz/zepeto-studio-kor#862] [5. Q&A]
빌드잇에서 작업한걸 유니티로 Map Loader 할때 관련해서 질문이 있습니다.

빌드잇에서 건물 외벽 작업만 진행하고 저장한뒤에 Map Loader 했을때 문제가 없는데

건물 내부도 빌드잇에서 작업진행하고 저장한뒤에 Map Loader 기능을 사용했을때 전에 건물 외벽만 올린 작업만 있더라구요.

혹시 Map Loader 할때 맵 기준이 어떻게 되는건지 알 수  있나요?

예를 들어 빌드잇 처음 저장했을때 작업만 불러올수 있다는지 이런 기준점이 있는지 알고 싶습니다.

(빌드잇에서 작업한 건물)
![물어볼것1](https://user-images.githubusercontent.com/107227678/172969562-ab26c7f5-43e1-4bc2-8aa7-642e67c9a525.JPG)

(유니티 Map Loader로 불러왔을때 건물 외벽만 있는 전에 저장했던 작업)
![물어볼것2](https://user-images.githubusercontent.com/107227678/172969715-7a706a5e-8731-444e-b7b5-728907d3c5fa.JPG)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 내부적으로 딜레이가 되는 부분을 개선해서 오늘 오전에 반영하였습니다.
빌드잇에서 최신으로 저장한 맵이 빌드잇 로더에 보이게 될 것입니다.
제보 감사 드립니다.

**Reply by heavybowgun** [reply]
감사합니다!
